detox/index.d.ts

```diff
@@ -1039,6 +1047,12 @@ declare global {
              */
             scrollTo(edge: Direction): Promise<void>;
 
+            /**
+             * Adjust slider to position.
+             * @example await element(by.id('slider')).adjustSliderToPosition(0.75);
+             */
+            adjustSliderToPosition(newPosition: number): Promise<void>;
```

Comment:
🤔  If you added this to `WaitForFacade`, could you please add the respective unit tests to [AndroidExpect.test.js](https://github.com/wix/Detox/blob/18026ce59e751da421e7c0d3f1b6f15ccaf1e988/detox/src/android/AndroidExpect.test.js) ?

Reply:
There is a test in androidExpect that handles this